Frontend/src/components/Layout/StyledModal.svelte

```diff
@@ -9,7 +9,7 @@
       props,
       {
         closeButton: false,
-        closeOnEsc: false,
+        closeOnEsc: true,
```

Comment:
Sorry to remove your code, but the library already has this built in functionality

Reply:
no need to feel sorry, perfect that it's already there :) !